gateway-service/src/main/resources/gateway-log-messages.yml

```diff
@@ -375,7 +375,61 @@ messages:
     # z/OSMF warning messages
     - key: org.zowe.apiml.security.auth.zosmf.serviceId
       number: ZWEAG181
-      type: WARNING
+      type: INFO
       text: "apiml.security.auth.zosmf.serviceId = '%s' is either not registered or not online yet."
       reason: "An incorrect value of the apiml.security.auth.zosmf.serviceId parameter is set in the configuration or it is not registered."
       action: "Ensure that the value of apiml.security.auth.provider is set either to 'dummy' if you want to use dummy mode, or to 'zosmf' if you want to use the z/OSMF authentication provider."
+
+    - key: org.zowe.apiml.security.auth.zosmf.sslError
+      number: ZWEAG182
+      type: ERROR
+      text: >
+        SSL Misconfiguration, z/OSMF is not accessible. Please verify the following: \n
```

Comment:
It can happen from the discovery point of view, but in the use of this message it can't because this is a direct call to z/OSMF, it doesn't need the trust relationship.